src/app/profile/settings/feature-opt-in/feature-opt-in.component.less

```diff
@@ -0,0 +1,25 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
```

Comment:
Not sure why I didn't catch this before, but why define the `font-family` for each element?

Reply:
fixed !